Spokestack/Spokestack.swift

```diff
@@ -30,25 +30,41 @@ import Foundation
     @objc public var configuration: SpeechConfiguration
     
     private var delegates: [SpokestackDelegate]
-    
+    private var editor: TranscriptEditor?
+
     /// This constructor is intended for use only by the `SpokestackBuilder`.
     /// - Parameters:
     ///   - delegates: Delegate implementations of `SpokestackDelegate` that receive Spokestack module events.
     ///   - configuration: Configuration properties for Spokestack modules.
     ///   - pipeline: An initialized SpeechPipeline for the client to access.
     ///   - nlu: An initialized NLUTensorflow for the client to access.
     ///   - tts: An initialized TextToSpeech for the client to access.
-    @objc internal init(delegates: [SpokestackDelegate], configuration: SpeechConfiguration, pipeline: SpeechPipeline, nlu: NLUTensorflow, tts: TextToSpeech) {
+    @objc internal init(delegates: [SpokestackDelegate], configuration: SpeechConfiguration, pipeline: SpeechPipeline, nlu: NLUTensorflow, tts: TextToSpeech, editor: TranscriptEditor?) {
         self.delegates = delegates
         self.configuration = configuration
         self.pipeline = pipeline
-        self.context = SpeechContext(configuration)
+        self.context = pipeline.context
         self.nlu = nlu
         self.tts = tts
+        self.editor = editor
         super.init()
     }
 }
 
+extension Spokestack: SpokestackDelegate {
+    
+    /// A required function for `SpokestackDelegate` implementors
+    /// - Parameter error: An error sent from a Spokestack module.
+    public func failure(error: Error) {}
+    
+    /// An event receiver used to fulfill automatic classification configuration.
+    /// - Parameter result: Global state for the speech pipeline.
+    public func didRecognize(_ result: SpeechContext) {
+        self.configuration.automaticallyClassifyTranscript ?
+            self.nlu.classify(utterance: self.editor?.editTranscript(transcript: result.transcript) ?? result.transcript) : nil
```

Comment:
IMO this would be more readable if `utterance` were broken out as its own variable.

Reply:
I really miss `->>`

Reply:
Ditto. RIP.